### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by froesef**

### CODE REVIEW

File: src/test/scala/com/ibm/m4d/mover/ConfigSuite.scala [hunk 1]
```diff
@@ -82,4 +82,36 @@ class ConfigSuite extends AnyFlatSpec with Matchers {
...
     substitutedConfig.hasPath("secretProviderRole") shouldBe false
   }
+
+  it should "parse DataFlowType correctly" in {
+    DataFlowType.parse("batch") shouldBe DataFlowType.Batch
+    DataFlowType.parse("Batch") shouldBe DataFlowType.Batch
+    DataFlowType.parse("stream") shouldBe DataFlowType.Stream
+    DataFlowType.parse("Stream") shouldBe DataFlowType.Stream
+    intercept[IllegalArgumentException](DataFlowType.parse("random"))
+  }
```
Comment (SeanRooooney) [inline src/test/scala/com/ibm/m4d/mover/ConfigSuite.scala:92]: Wouldn't  be better to normalize the input, i.e. put everything lower case ?

**Review by SeanRooooney — APPROVED** [review]
Reading the code and thinking about the structure. I wonder of Iceberg is really a file format, or a different type of data store

File: src/test/scala/com/ibm/m4d/mover/ConfigSuite.scala (hunk 1 above)
Comment (froesef) [reply src/test/scala/com/ibm/m4d/mover/ConfigSuite.scala:92]: `DataFlowType.parse` is putting it to lower case before doing the comparison. This tests if it's also implemented that way.

**Comment by froesef** [comment]
You're right. I think Iceberg etc may actually be treated as different data stores as there will probably also be a different treatment of key/value (when coming as stream from Kafka) than we would have in a normal file.

**Pull request merged**